testproject/.gitignore

```diff
@@ -69,3 +69,5 @@ crashlytics-build.properties
 # Temporary auto-generated Android Assets
 /[Aa]ssets/[Ss]treamingAssets/aa.meta
 /[Aa]ssets/[Ss]treamingAssets/aa/*
+
+InitTestScene*
```

Comment:
where is this coming from?

Reply:
Test Runner doesn't cleanup after itself when running playmode tests if there's a failure during the test. So you endup with tmp scenes accumulating. This is to stay sane while doing a git status :P